build.sbt

```diff
@@ -7,11 +7,17 @@ lazy val `course-management-tools` =
       delinearize,
       mainadm
     )
-    .settings(ThisBuild / scalaVersion := Version.scalaVersion)
+    .settings(CommonSettings.commonSettings: _*)
+    .settings(skip in publish := true)
```

Comment:
I have noticed that removing `.settings(ThisBuild / scalaVersion := Version.scalaVersion)` in another project (a Dotty project, to be precise), caused `scalaVersion` not to be picked up in sub-projects. You can see that when starting a REPL session in a sub-project.

Reply:
`scalaVersion` is applied to all sub projects as follows:
```
sbt:course-management-tools> scalaVersion
[info] studentify / scalaVersion
[info] 	2.13.3
[info] mainadm / scalaVersion
[info] 	2.13.3
[info] delinearize / scalaVersion
[info] 	2.13.3
[info] linearize / scalaVersion
[info] 	2.13.3
[info] core / scalaVersion
[info] 	2.13.3
[info] scalaVersion
[info] 	2.13.3
```
I can confirm this doesn't affect the course templates and is specific to some dotty projects